powerauth-push-server/pom.xml

```diff
@@ -175,11 +182,6 @@
             <version>${bc.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-tomcat</artifactId>
-            <scope>test</scope>
```

Comment:
One more thing. In the parent pom, we have excluded `tomcat-embed-el` and even `spring-boot-starter-tomcat`.